alert-common/src/main/java/com/synopsys/integration/alert/common/rest/ResponseFactory.java

```diff
@@ -75,6 +75,18 @@ public static ResponseStatusException createNotImplementedException(@Nullable St
         return new ResponseStatusException(HttpStatus.NOT_IMPLEMENTED, customMessage);
     }
 
+    public static <T> ResponseStatusException createStatusException(ActionResponse<T> actionResponse) {
+        String customMessage = actionResponse.getMessage().orElse("");
```

Comment:
In `ResponseStatusException`, the message parameter in `@Nullable`. We should use `.orElse(null)` here so that the default message for the `HttpStatus` will be used by the exception.